pages/guides/receive-money/create-a-customer.mdx

```diff
@@ -99,20 +101,20 @@ print($customer); # => "https://api-sandbox.dwolla.com/customers/c7f300c0-f1ef-4
 ?>
 ```
 
-<ol class="alerts">
-  <li class="alert icon-alert-alert">
-    Provide the IP address of the end-user accessing your application as the
-    `ipAddress` parameter. This enhances fraud detection and tracking.
-  </li>
-</ol>
+<AlertBar variation="info">
+  Providing the IP address of the end-user accessing your application as the
+  <code>ipAddress</code> parameter. This enhances fraud detection and tracking.
+</AlertBar>
```

Comment:
When components are used back-to-back (with no text between them) they scrunch together. In this case, it's the code block above and this alert bar. This isn’t necessarily something we have to fix ahead of the internal launch, but we should discuss and fix it ahead of public launch

Reply:
Yeah sounds good. I've seen using a `<br/>` tag in between works to an extent, but we may have to think through either adding a wrapper div around it, or something in the MDX styling that adds padding to JSX components.

Reply:
Adding `<br/>` tags sounds like a good quick fix though! Thanks for calling that out, Shreya!